cmake/arrow/CMakeLists.txt

```diff
@@ -277,11 +277,7 @@ target_compile_definitions(arrow PUBLIC ARROW_STATIC)
 
 # will need built boost filesystem and system .lib to work, even though
 # perspective itself does not use those dependencies
-target_link_libraries(arrow
-    ${double-conversion_LIBRARIES}
-    ${Boost_FILESYSTEM_LIBRARY}
-    ${Boost_SYSTEM_LIBRARY}
-    ${ARROW_TEST_LINK_TOOLCHAIN})
+target_link_libraries(arrow)
```

Comment:
This doesn't do anything anymore, are we sure `double-conversion` isn't required?